storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2998,6 +3160,11 @@ const char *xb_get_copy_action(const char *dflt) {
 /* TODO: We may tune the behavior (e.g. by fil_aio)*/
 
 static bool xtrabackup_copy_datafile(fil_node_t *node, uint thread_n) {
+  return xtrabackup_copy_datafile(node, thread_n, nullptr);
```

Comment:
lets call the overloaded function by idfferent name. may be add _func or _low().

Reply:
AB